view/theme/frio/style.php

```diff
@@ -44,65 +79,62 @@
 if (! $schema) {
 	if(file_exists('view/theme/frio/schema/default.php')) {
 		$schemefile = 'view/theme/frio/schema/default.php';
-		require_once ($schemefile);
+		require_once $schemefile;
 	}
 	if(file_exists('view/theme/frio/schema/default.css')) {
-		$schemecss = file_get_contents('view/theme/frio/schema/default.css');
+		$schemecssfile = 'view/theme/frio/schema/default.css';
 	}
 }
 
 //Set some defaults - we have to do this after pulling owner settings, and we have to check for each setting
 //individually.  If we don't, we'll have problems if a user has set one, but not all options.
-if(! $nav_bg)
-	$nav_bg = "#708fa0";
-if(! $nav_icon_color)
-	$nav_icon_color = "#fff";
-if(! $link_color)
-	$link_color = "#6fdbe8";
-if(! $bgcolor)
-	$bgcolor = "#ededed";
-if(! $contentbg_transp)
-	$contentbg_transp = 100;
-if(! $background_image)
-	$background_image ='';
-
-// Calculate some colors in dependance of existing colors
+$nav_bg           = (empty($nav_bg)           ? "#708fa0"      : $nav_bg);
```

Comment:
Just as an aside, you can write default values more easily with ternary operators:

```php
$value = $value ? : 'default';
```

Reply:
But wouldn't this make a problem when ```$nav_bg``` was undeclared?

Reply:
Yes it would, adding an `isset()` check breaks the magic of the two-parameters ternary operator. If `$nav_bg` is potentially undeclared, here's the correct form:
````php
$value = isset($value) && $value ? $value : 'default';
````

Reply:
An ```isset($value) && $value``` is the same as ```!empty($value)```

Reply:
but `empty($value)` is the same like `!isset($value) && !$value`

Reply:
So the lines above should be written this was:
```
$value = !empty($value) ? : 'default';
```
Correct?

Reply:
> but `empty($value)` is the same like `!isset($value) && !$value`

Fair enough, I didn't know. Thanks for the info!

@annando No, the magic of `$value ? : 'default'` is to use the result of the test before the `?` as the first value before the `:`. If you use a boolean test like `empty()`, the result will be either  `true` or `false`, not the value of the tested variable.